exchanges/bithumb/bithumb_wrapper.go

```diff
@@ -122,8 +123,93 @@ func (b *Bithumb) UpdateOrderbook(p pair.CurrencyPair, assetType string) (orderb
 // GetAccountInfo retrieves balances for all enabled currencies for the
 // Bithumb exchange
 func (b *Bithumb) GetAccountInfo() (exchange.AccountInfo, error) {
-	var response exchange.AccountInfo
-	return response, errors.New("not implemented")
+	var info exchange.AccountInfo
+	info.ExchangeName = b.GetName()
+
+	bal, err := b.GetAccountBalance("ALL")
+	if err != nil {
+		return info, err
+	}
+
+	var exchangeBalances []exchange.AccountCurrencyInfo
+
+	// Added due to increasing of the usuable currencies on exchange, usually
+	// without notificatation, so we dont need to update structs later on
+	for tag, datum := range bal.Data {
+		splitTag := common.SplitStrings(tag, "_")
+		c := splitTag[len(splitTag)-1]
+		switch splitTag[0] {
+		case "available":
+			// NOTE: Captures available data
+
+		case "in":
+			var hold float64
+			if reflect.TypeOf(datum).String() != "float64" {
+				hold, err = strconv.ParseFloat(datum.(string), 64)
+				if err != nil {
+					return info, err
+				}
+			} else {
+				hold = datum.(float64)
+			}
+
+			var appended bool
+			for i := range exchangeBalances {
+				if exchangeBalances[i].CurrencyName == c {
```

Comment:
👍

Reply:
Actually though, to your point above, you'd want to skip any split tag that isn't "in" and "total", since my code considers things way later. So something like 
`splitTag := common.SplitStrings(tag, "_")
if splitTag != "in" || splitTag != "total" {
continue 
}`